frontend/src/components/spin.tsx

```
@@ -0,0 +1,25 @@
+import type { FC } from "react";
+
+const Spin: FC<IComponentProps<HTMLOrSVGElement>> = ({ className, ...restProps }) => {
+  return (
+    <div className="flex h-screen w-full flex-col items-center">
+      <svg
+        className={clsx("h-6 w-6 animate-spin text-green-500", className)}
+        xmlns="http://www.w3.org/2000/svg"
+        fill="none"
+        viewBox="0 0 24 24"
+        {...restProps}
+      >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)